server/src/main/java/org/opensearch/index/translog/Translog.java

```diff
@@ -138,6 +138,7 @@ public abstract class Translog extends AbstractIndexShardComponent implements In
     protected final BigArrays bigArrays;
     protected final ReleasableLock readLock;
     protected final ReleasableLock writeLock;
+    protected final Object uploadMutex = new Object();
```

Comment:
Translog is not doing any upload per se.

Reply:
Removed this.